include/cantera/zeroD/flowControllers.h

```diff
@@ -84,26 +94,57 @@ class Valve : public FlowDevice
 public:
     Valve();
 
+<<<<<<< HEAD
     virtual std::string typeStr() const {
         return "Valve";
     }
 
     virtual bool ready() {
         return FlowDevice::ready() && (m_coeffs.size() == 1 || m_func);
+=======
+    //! Set the proportionality constant between pressure drop and mass flow
+    //! rate
+    /*!
+     * *c* has units of kg/s/Pa. The mass flow rate is computed as:
+     * \f[\dot{m} = c \Delta P \f]
+     */
+    //! @deprecated To be removed after Cantera 2.5.
+    void setPressureCoeff(double c) {
+        warn_deprecated("Valve::setParameters()",
+                        "To be removed after Cantera 2.5. "
+                        "Use Valve::setValveCoeff instead.");
+        m_coeff = c;
+>>>>>>> bfab8a070... [Reactor] clarified FlowDevice interface
     }
 
     //! Set the proportionality constant between pressure drop and mass flow
     //! rate
     /*!
      * *c* has units of kg/s/Pa. The mass flow rate is computed as:
+     * \f[\dot{m} = c l(t) f(\Delta P) \f]
+     * where *l* and *f* are functions of time and pressure drop that are set
+     * by `setValveLift` and `setFunction`, respectively. If no functions are
+     * specified, the mass flow rate defaults to:
      * \f[\dot{m} = c \Delta P \f]
      */
-    void setPressureCoeff(double c) {
-        m_coeffs = {c};
+    void setValveCoeff(double c) {
+        m_coeff = c;
+    }
+
+    //! Get the valve coefficient.
+    double getValveCoeff() {
+        return m_coeff;
     }
 
+    //! Set the timing for a Valve object. If provided, the function is evaluated
+    //! as a function of time..
+    void setValveLift(Func1* f);
```

Comment:
I would like to avoid the terminology "lift" which is pretty specific to the poppet valves used in IC engines. Perhaps "opening" would be a more general alternative, i.e. this could be `setOpeningFunction`. Or calling it `setTimeFunction` would provide a parallel to naming f(dP) the `pressureFunction`.

Reply:
`setTimeFunction` and `setPressureFunction` are excellent suggestions.